src/modules/auth/auth_http.ts

```diff
@@ -0,0 +1,23 @@
+import { Router } from 'express'
+import rateLimit from 'express-rate-limit'
+import { validate } from '../../helpers/validator'
+import { verifyAccessToken } from '../../middleware/jwt'
+import { Auth as Rules } from './auth_rules'
+import { Auth as Handler } from './auth_handler'
+import config from '../../config'
+
+const apiLimiterSignIn = rateLimit({
+  windowMs: Number(config.get('api.limiter.time.signin', 300000)), // Default time signin 5 minutes
+  max: Number(config.get('api.limiter.max.signin', 3)),
+});
+
+const router = Router()
+
+router.post('/v1/auth/users/sign-up', validate(Rules.signUp), Handler.signUp)
```

Comment:
better ini `'/v1/auth/users/` kalo bisa pake router group / prefix sih jadi lebih simpel

Reply:
bisa aja mang, berarti di tambahinnya saat diuse di server.ts, 
jadi seperti ini, `this.app.use('/v1/auth/users', auth)`
![image](https://user-images.githubusercontent.com/41193120/145504073-97097704-eb7e-445f-b60c-c9a285714693.png)

Reply:
https://expressjs.com/en/4x/api.html#router

Reply:
oh yaya,, is oke atuh,, bener sih gitu aja kalo ingin jadi dinamic toute mah 👍